csharpcore/GildedRoseTests/GildedRoseTests.csproj

```diff
@@ -8,9 +8,8 @@
 
     <ItemGroup>
         <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.0.0" />
-        <PackageReference Include="xunit" Version="2.4.1" />
-        <PackageReference Include="xunit.runner.visualstudio" Version="2.4.3" />
         <PackageReference Include="coverlet.collector" Version="3.1.0" />
+        <PackageReference Include="NUnit" Version="3.13.3" />

```

Comment:
That's how a project (.csproj) reference a nugget.
It's change automatically in Test -> Nugget Packages
You can update nuggets with the Nugget Package Manager in your IDE normally